components/avtar/avtar.css

```diff
@@ -0,0 +1,83 @@
+/* Temp Class */
+.avtar-wrapper {
```

Comment:
Please correct the avatar spelling. It is correct in your example heading for image and text avatar.

Reply:
Done